packages/astro/src/integration/index.ts

```diff
@@ -79,3 +84,17 @@ function findDefaultSdkInitFile(type: 'server' | 'client'): string | undefined {
     .map(ext => path.resolve(path.join(process.cwd(), `sentry.${type}.config.${ext}`)))
     .find(filename => fs.existsSync(filename));
 }
+
+function getSourcemapsAssetsGlob(config: AstroConfig): string {
+  // outDir is stored as a `file://` URL
+  const outDirPathname = config.outDir && config.outDir.pathname;
+  const rootDirName = config.root && config.root.pathname;
+
+  if (outDirPathname && rootDirName) {
+    const relativePath = path.relative(rootDirName, outDirPathname);
+    return `${relativePath}/**/*`;
+  }
+
+  // fallback to default output dir
+  return 'dist/**/*';
```

Comment:
technically, these options should always be set by Astro, even if users didn't set them explicitly in their config file. However, I think it makes sense to still fall back to a safe case in case anything went wrong before our integration came in.